Added clearer instructions to the "Create Many Records with model.create()" challenge (#38026)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-many-records-with-model.create.english.md

## Instructions
<section id='instructions'>

> Add a function expecting an <code>arrayOfPeople</code> argument which creates many people through <code>Model.create()</code>.

Reviewer: ```suggestion
Modify the <code>createManyPeople</code> function to create many people through <code>Model.create()</code> and the argument <code>arrayOfPeople</code>.
```
The function already exists. If a user creates a function with a different name, they would need to modify code in sections we've told them not to touch.

> Modify the <code>createManyPeople</code> function to create many people through <code>Model.create()</code> and the argument <code>arrayOfPeople</code>.

Reviewer 2: ```suggestion
Modify the <code>createManyPeople</code> function to create many people using <code>Model.create()</code> with the argument <code>arrayOfPeople</code>.
```

> (You can reuse the model you instantiated in the previous exercise)

Reviewer 2: ```suggestion
<strong>Note: </strong> You can reuse the model you instantiated in the previous exercise.
```
